public/app/controllers/subjects_controller.rb

```diff
@@ -61,7 +61,18 @@ def index
       @sort_opts.unshift(all_sorts['relevance'])
     end
     @no_statement = true
-    render 'search/search_results'
```

Comment:
Is this entire controller overwritten in `plugins/aspace-doris-public/public/controllers/subjects_controller.rb`? If so then we can revert the changes back to original here.